stories/components/teamsChannelPicker/teamsChannelPicker.stories.js

```diff
@@ -84,6 +84,38 @@ export const selectChannel = () => html`
   </script>
 `;
 
+export const clearSelectedItem = () => html`
+  <mgt-teams-channel-picker></mgt-teams-channel-picker>
+
+  <button class="get">Get SelectedChannel</button>
+  <button class="clear">Clear SelectedChannel</button>
+
+  <div class="output"></div>
+
+  <script>
+    document.querySelector('.get').addEventListener('click', _ => {
+      const picker = document.querySelector('mgt-teams-channel-picker');
+      const output = document.querySelector('.output');
+
+      if (picker.selectedItem) {
```

Comment:
perhaps this logic should be run when the selectionChanged event is fired by the picker component rather than as a button click.

Reply:
This test is specifically testing that `clearSelectedItem()` method does clear the selected channel from the picker, and for this I'll still need to target this method via the button.

Reply:
Yeah, my comment was specifically about updating the content after invoking the clearSelectedItem method in the click handler, looks like you landed on the solution I was after when you made the changes :)